content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -240,14 +240,13 @@ npx prisma db pull
 The resulting `view` block will be defined as follows:
 
 ```prisma
-/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+/// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by Prisma Client.
```

Comment:
This quotes an error message - if the error message is grammatically incorrect, this has to be fixed in the actual error message code, not just where we quote it.